public/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta property="og:title" content="Balance Tetris">
+    <meta property="og:description" content="Tetris built with vanilla javascript">
+    <meta property="og:image" content="https://raw.githubusercontent.com/WoochanLee/BalanceTetris/gh-pages/../res/image_800x800.png">
+    <meta name="description" content="Tetris built with vanilla javascript">
+    <meta name="keywords" content="Tetris,balancehero,balancetetris,테트리스,밸런스히어로,밸런스테트리스">
+    <link rel="shortcut icon" type="image/x-icon" href="../res/favicon.ico">
+    <link rel="stylesheet" href="../css/screen.css">
+    <title>Balance Tetris</title>
+    <style>
+      
+    </style>
+  </head>
+  <body>
+    <div class="main">
+      <!-- <button id="test" onclick="onClickTestButton();">스탑</button> -->
+      <!-- <canvas id="canvas" width="295" height="745"></canvas> -->
+      <!-- <h1>BALANCE TETRIS</h1> -->
```

Comment:
I guess here are some test codes.
Please consider erasing here if not required.